added input component

tusharkandpal opened the pull request:
- input type: simple & validation form
- styles for both types

tusharkandpal committed:
added input component

- input type: simple & validation form
- styles for both types

junaidshaikh-js commented on components/input/input.css:
```diff
@@ -0,0 +1,102 @@
+  margin: 0.5rem;
+  min-width: 6rem;
+}
+.required:after {
```
use double colon for pseudo element

tusharkandpal replied:
Ya will make the change. Thanks!

junaidshaikh-js reviewed (commented):
Looks good to me!

shashidwivedi commented on components/input/input.html:
```diff
@@ -0,0 +1,218 @@
+            feedback /comments, etc.
+          </p>
+          <div class="component-display mp-m">
+            <form class="input-form">
```
You can name this class name as form-container or something, because input can be confused with field in CSS.

tusharkandpal replied:
Yes will change it! Thanks @shashidwivedi

shashidwivedi commented on components/input/input.html:
```diff
@@ -0,0 +1,218 @@
+              <div class="form-field">
+                <label class="form-label required" for="name">Username </label>
+                <input
+                  class="form-input-field"
```
See, here you have used 'input' word for an input element. If this is the way you want it to use it. Please change the parent form class name as suggested above. In this way, you can simply use form-input here, instead of form-input-field

tusharkandpal committed:
refactored component

- changed `form-input` to `form-container`
- also, `form-input-field` to `form-input`
- updated iframe snippets

tusharkandpal committed:
corrected pseudo element `after`

tusharkandpal commented:
did the changes mentioned by you both. please approve! @junaidshaikh-js @shashidwivedi
